skimage/io/collection.py

```diff
@@ -416,39 +416,36 @@ class MultiImage(ImageCollection):
 
     Notes
     -----
-    The object that is returned can be used as a list of image-data objects,
-    where each entry in the list represents one image. In this regard it is
-    very similar to `ImageCollection`, but the two differ in the treatment
-    of multi-frame images.
+    The object that is returned is a list of image-data arrays. In this
```

Comment:
What if:
```suggestion
    `MultiImage` returns a list of image-data arrays. In this
```
Then the other sentence (`In this regard, it is...`) would make more sense, I guess.

Reply:
You're absolutely right, thank you!